python/ray/tune/BUILD

```diff
@@ -682,7 +682,7 @@ py_test(
     size = "large",
     srcs = ["examples/pbt_transformers/pbt_transformers.py"],
     deps = [":tune_lib"],
-    tags = ["flaky", "exclusive", "example"],
+    tags = ["exclusive", "example"],
```

Comment:
this change intentional? dropped tune test from the flaky list

Reply:
It wasn't passing after the RLlib install command (some dependency issue), unfortunately.

It seems one problem with the flaky test builder is it won't work for certain tests that require specific dep setups.